docs/pages/guides/tutorials/Kubernetes-deployments.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Exploring Kubernetes Deployments with Meshery Playground
+abstract: Learn Kubernetes Pods
+permalink: guides/tutorials/kubernetes-deployments
+model: kubernetes
+kind: deployments
+type: guides
+category: tutorials
+language: en
+list: include
+abstract: "Explore Kubernetes Deployments with Service using Meshery in an interactive live environment, through a series of hands-on exercises."
+---
+
+## Introduction
+In this tutorial, we will delve into the world of Kubernetes Deployments, a key resource for managing and scaling containerized applications. Utilizing Meshery Playground, an interactive live cluster environment, we'll conduct hands-on labs to gain practical experience in deploying and managing applications using Kubernetes Deployments.
+
+### Prerequisites
+- Basic understanding of Kubernetes concepts.
+- Meshery Playground access. If you don't have an account, sign up at [Meshery Playground](https://meshery.layer5.io/play).
+
+
+#### creating a standalone deployment
+ Here is the deployment.yaml we are going use for this tutorial 
+
+ ``` yaml
+ apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  labels:
+    app: nginx
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+```
+
+
+   1. In the MeshMap screen, rename the design from _Untitled Design_ to a name of choice. This helps in identifying the design later easily.
+   2. Click **Components** tab.
+   ![](./screenshots/1.png)
+   3. Search for **deployment** in the list of components and choose only kubenetes deployment with the bellow symbol .
+   ![](./screenshots/2%20.png)
+
+   4. Scroll down, _select and drag_ the **deployment** component from the search results to the design canvas on the right.
+   5. After successfully drop the **deployment** component into canvas ,double click on deployment to enter configurations.
+   6. Rename the **deployment** to **nginxdeployment**  or any desired name .
+   ![](./screenshots/3 and 4.png)
+
+### Setting minimum no of replicas for autohealing capability
+
+   7. In the Spec section , U  will have an option to define **replicas**, fill that with ur desired number of replicas and  for this tutorial we can use three replicas as the base minimum .
```

Comment:
IMO, since this is docs for users, it seems more appropriate to use `you` wherever you use `u` or `U`.